### PR TITLE
Add ftfy 4.4.2.

### PR DESCRIPTION
**Pull request by rolando**
This is required for https://github.com/conda-forge/spacy-feedstock/pull/10/ and related to https://github.com/conda-forge/staged-recipes/pull/2455

### CODE REVIEW

File: recipes/ftfy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    - ftfy.fixes
+    - ftfy.formatting
+
+  commands:
+    - ftfy -h
+
+about:
+  home: https://github.com/LuminosoInsight/python-ftfy/
+  license: MIT
+  summary: Fixes some problems with Unicode text after the fact
```
Comment (synapticarbors) [inline recipes/ftfy/meta.yaml]: please include:

```
license_file: LICENSE.txt
...
dev_url: https://github.com/LuminosoInsight/python-ftfy
doc_url: http://ftfy.readthedocs.io/
```

File: recipes/ftfy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
+{% set name = "ftfy" %}
+{% set version = "4.4.2" %}
```
Comment (synapticarbors) [inline recipes/ftfy/meta.yaml]: It appears as if the newest version is `5.1` on pypi. I think it's not compatible with the version of spacy that's attempting to be packaged. Once this is accepted, please be sure to update the recipe to keep up-to-date with the package's releases.

Comment (rolando) [inline recipes/ftfy/meta.yaml]: Yes, I'm playing safe adding the minimum version required and then will update to the latest `5.x`. Also, `ftfy 5.x` is py3 only.

File: recipes/ftfy/meta.yaml (hunk 1 above)
Comment (rolando) [inline recipes/ftfy/meta.yaml]: This version does not include the `LICENSE.txt` file in the dist file. Development version does.

I've added the other url fields.

Comment (rolando) [inline recipes/ftfy/meta.yaml]: My bad, actually 4.4.2 does contain the license file. The previous versions didn't.

**Pull request merged**